starlark/eval_test.go

```diff
@@ -764,6 +764,16 @@ func TestUnpackRequiredAfterOptional(t *testing.T) {
 	}
 }
 
+func TestUnpackSuggestsKwargNameBasedOnSpelling(t *testing.T) {
```

Comment:
This test could be expressed in two lines in testdata/function.star:

```
def misspelled(response): None
misspelled(reponse=1) ### ...did you mean...
```

Reply:
I am missing something here. How can this test be able to suggest the correct spelling?

My attempt fails:
```python
---
# Unpacking kwargs should suggest keyword name based on spelling
load("assert.star", "assert")

def f():
    def misspelled(response): None
    misspelled(reponse=1)

assert.fails(f, ".+did you mean response?.+")

# --- FAIL: TestExecFile (0.12s)
#     starlarktest.go:117: Traceback (most recent call last):
#           testdata/function.star:333:13: in <toplevel>
#           builtins/assert.star:39:14: in _fails
#         Error: regular expression (.+did you mean response?.+) did not match error (function misspelled got an unexpected keyword argument "reponse")
```

Reply:
Ah, of course: it has to be a a built-in function. Try:

```
---
# Unpack provides spell check for argument names.
min([], keg=1) ### `did you mean key\?`
```